var/spack/repos/builtin/packages/samtools/package.py

```diff
@@ -33,16 +33,20 @@ class Samtools(Package):
     homepage = "www.htslib.org"
     url = "https://github.com/samtools/samtools/releases/download/1.3.1/samtools-1.3.1.tar.bz2"
 
+    version('1.7', '2240175242b5183bfa6baf1483f68023')
     version('1.6', 'b756f05fd5d1a7042074417edb8c9aea')
     version('1.4', '8cbd7d2a0ec16d834babcd6c6d85d691')
     version('1.3.1', 'a7471aa5a1eb7fc9cc4c6491d73c2d88')
     version('1.2', '988ec4c3058a6ceda36503eebecd4122')
 
     depends_on('ncurses')
     # htslib became standalone @1.3.1, must use corresponding version
+    depends_on('htslib@1.7',   when='@1.7')
     depends_on('htslib@1.6',   when='@1.6')
     depends_on('htslib@1.4',   when='@1.4')
     depends_on('htslib@1.3.1', when='@1.3.1')
+    depends_on('zlib', when='@1.7:')
+    depends_on('bzip2', when='@1.7:')
```

Comment:
These definitely weren't dependencies before?

Reply:
no, confirmed I could install 1.6 without, but not 1.7

Reply:
also, 1.8 seems to have added a dependency on lzma, but I haven't finished that yet